packages/core/src/components/validation-tooltip/validation-tooltip.tsx

```diff
@@ -75,7 +75,9 @@ export class ValidationTooltip {
   }
 
   private destroyAutoUpdate() {
-    this.tooltipElement.style.display = 'none';
+    if (this.tooltipElement) {
+      this.tooltipElement.style.display = 'none';
+    }
```

Comment:
More concise solution?

```suggestion
    this.tooltipElement?.style.display = 'none';
```

Reply:
The more concise solution is not possible in this case.

![image](https://github.com/siemens/ix/assets/151610373/917f51e4-a812-4cf5-b4ba-0fcf53d9e9bc)